docs/docs/guides/how-to/cms-integration-plugin/index.mdx

```diff
@@ -0,0 +1,1988 @@
+---
+title: 'Building a CMS Integration Plugin'
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+A CMS integration plugin allows you to automatically synchronize your Vendure product catalog with an external Content Management System.
+
+This is done in a way that establishes Vendure as the source of truth for the entities' data.
```

Comment:
I think `entities data` is very generic - let's be more specific here, like "ecommerce data" or something else.